client/users.html

```diff
@@ -22,7 +22,7 @@
   <button type="button" id="getUsers">Get Users</button>
 
 
-  <pre id="jsonDump" > Json will go here</pre>
+  <pre id="jsonDump"> JSON will go here</pre>
```

Comment:
Is the space before JSON something we did on purpose?

```suggestion
  <pre id="jsonDump">JSON will go here</pre>
```